test/functional/rpc_gettransaction.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test gettransaction RPC command"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class GetTransactionTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 1
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        self.test_gettransaction_decoding()
+    
+    def test_gettransaction_decoding(self):
+        self.log.info("Testing gettransaction decoding...")
+        self.nodes[0].generate(101)
+        txid = self.nodes[0].sendtoaddress(self.nodes[0].getnewaddress(), 1)
+        # False for 'include_watchonly', True for 'decode'
+        tx = self.nodes[0].gettransaction(txid, False, True)
```

Comment:
You can use named arguments and drop the comment above (untested):
```py
tx = self.nodes[0].gettransaction(txid=txid, decode=True)
```

Reply:
Yes, tested it it works thanks :D